UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/InputPointerVisualizer.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
```

Comment:
Please remove the `Demos.EyeTracking` from the namespace

Nit Pick/Style....Please start fields with a lowercase letter, not underscore.